integrations/katalon_integration_demo_project /codemagic.yaml

```diff
@@ -0,0 +1,67 @@
+# Check out https://docs.codemagic.io/getting-started/yaml/for more information
+
+workflows:
+  native-android:
+    name: Native Android
+    instance_type: mac_pro
+    environment:
+      groups:
+        # Add the group environment variables in Codemagic UI (either in Application/Team variables) - https://docs.codemagic.io/variables/environment-variable-groups/
+        - keystore_credentials # <-- (Includes CM_KEYSTORE, CM_KEYSTORE_PASSWORD, CM_KEY_ALIAS_PASSWORD, CM_KEY_ALIAS_USERNAME)
+          # CM_KEYSTORE <-- Put your keystore file here
+          # CM_KEYSTORE_PASSWORD <-- Put your keystore password here
+          # CM_KEY_ALIAS_PASSWORD <-- Put your keystore alias password here
+        # CM_KEY_ALIAS_USERNAME <-- Put your keystore alias username here
+        - google_play # <-- (Includes GCLOUD_SERVICE_ACCOUNT_CREDENTIALS)
+        - other
+      vars:
+        # Set up paths to application binary and test package
+      node: v16.11.1
+      xcode: latest
+    triggering:
+      events:
+        - push
+        - tag
+        - pull_request
+      branch_patterns:
+        - pattern: develop
+          include: true
+          source: true
+    scripts:
+      - name: Set Android SDK location
+        script: |
+          echo "sdk.dir=$ANDROID_SDK_ROOT" > "$CM_BUILD_DIR/local.properties"
+      - name: Set up keystore
+        script: |
+          echo $CM_KEYSTORE | base64 --decode > /tmp/keystore.keystore
+          # The following script creates key.properties file and stores the credentials in it. As we configure code signing in app/build.gradle file, the following part is unnecessary unless code signing is configured differently as explained in the documentation: https://docs.codemagic.io/flutter-code-signing/android-code-signing/
+          # cat >> "$CM_BUILD_DIR/key.properties" <<EOF
+          # storePassword=$CM_KEYSTORE_PASSWORD
+          # keyPassword=$CM_KEY_ALIAS_PASSWORD
+          # keyAlias=$CM_KEY_ALIAS_USERNAME
+          # storeFile=/tmp/keystore.keystore
+          # EOF
```

Comment:
Code samples should encourage the use of code signing identities for Android so separate environment variables don't have to be added and no extra script step is required for decoding the keystore.

Reply:
Code signing identities is a Team feature, should we include it in our sample projects, considering free account users will use this as well ?